src/DETHRACE/common/displays.c

```diff
@@ -1204,31 +1204,31 @@ void DoInstruments(tU32 pThe_time) {
             }
 
             if (the_angle >= 0.0) {
-                if (the_angle >= 6.283185307179586) {
-                    the_angle = the_angle - 6.283185307179586;
+                if (the_angle >= TAU) {
+                    the_angle = the_angle - TAU;
                 }
             } else {
-                the_angle = the_angle + 6.283185307179586;
+                the_angle = the_angle + TAU;
             }
-            the_angle2 = 1.570796326794897 - the_angle;
+            the_angle2 = DR_PI_OVER_2 - the_angle;
```

Comment:
Personally, I would toss the `DR_` prefix. We're not using it for `PI` and `TAU` either.

Reply:
The DR prefix is to differentiate it from `PI` as defined in brender defs (they have different number of decimal points). This way it is easy to check floating calculations between original game and dethrace because the same constant value is used regardless of platform.